apispec/openapi.json

```diff
@@ -5586,7 +5586,7 @@
       "Accept": {
         "name": "Accept",
         "in": "header",
-        "description": "Indicates the response format, supports \"application/json\" and \"application/xml\", default value is \"application/json\".",
+        "description": "Indicates the response format, supports \"application/json\" and \"application/xml\". The default value is \"application/json\".",
```

Comment:
Suggestion:   "Indicates the response format, either \"application/json\" or \"application/xml\".